rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/MavenExecutor.java

```diff
@@ -75,15 +86,27 @@ public int executeInNewJvm(String parameters, String pathToPom, String testParam
 		command.add(jvm);
 		command.add("-cp");
 		command.add(getClassPath());
+		Properties props = System.getProperties();
+		for (String key : props.stringPropertyNames()) {
+			if (key.startsWith("http.")) {
+				command.add("-D" + key + "=" + props.getProperty(key));
+			}
+		}
+		String mavenSettings = System.getProperty("TE.MAVENSETTINGSPATH");
+		if (mavenSettings != null) {
+			if (mavenSettings.length() > 0) {
+				command.add("-DTE.MAVENSETTINGSPATH="+mavenSettings);
+			}
+		}
 		command.add(this.getClass().getName());
 		command.add(parameters);
 		command.add(pathToPom);
 		command.add(testParam);
-		logger.trace("Execute maven in new jvm with: {}", command);
 		ProcessBuilder processBuilder = new ProcessBuilder();
 		processBuilder.inheritIO();
 		processBuilder.directory(new File(pathToPom));
 		processBuilder.redirectErrorStream(true);
+		logger.info("Execute maven in new jvm with: {}", command);
```

Comment:
I would prefer "Executing". Also our logging conventions propose to write the statement like this: `... in new jvm with command='{}'."`